lib/std/os/bits/linux/arm64.zig

```diff
@@ -82,6 +82,7 @@ pub const SYS_pread64 = 67;
 pub const SYS_pwrite64 = 68;
 pub const SYS_preadv = 69;
 pub const SYS_pwritev = 70;
+pub const SYS_sendfile = 71;
```

Comment:
o.o how odd that this was missed!

Reply:
yeah... i made some confused noises out loud when I saw that, lol